# Defines a object with associated region in 3D space

# Header of the message
std_msgs/Header header

# The detected 3D objects with associated regions
vision_msgs/Detection3D object

# The name of the region associated with the object
string region